global_config.py

```diff
@@ -11,23 +11,23 @@
 MAILGUN_NAME = 'mg.hostname.com'
 MAILGUN_SENDER = 'noreply@mg.hostname.com'
 
-# URL of your Druid instance 
-# Druid is a real time analytics data base. We use it to store all of our 
+# URL of your Druid instance
+# Druid is a real time analytics data base. We use it to store all of our
 # analytical data (i.e. data that can be queried in AQT).
-DEFAULT_DRUID_HOST = 'https://druid.hostname.com'
+DEFAULT_DRUID_HOST = 'http://ideonvn.asuscomm.com'
```

Comment:
We should change this back to a dummy URL since it's a file that all users will need to copy and update.

Reply:
Can you also revert the changes to `PASSPHRASE_ENDPOINT`, `NOREPLY_EMAIL`, `SUPPORT_EMAIL`, `RENDERBOT_EMAIL`, and `DATA_UPLOAD_DEFAULT_NOTIFY`? We don't want Harmony instances to provide Zenysis addresses.

Reply:
Lastly `REDIS_HOST` should be reverted before we land this PR. We'll have to add instructions for setting all of this up to the Harmony docs.

Reply:
The idea here is more for your testing, without having to fiddle with these, but once the PR is ready to merge this will be removed since it is unique to testing the instance.

However I'll remove this and base another PR of this to have these in there.